conduit/src/test/java/io/rtr/conduit/integration/ShutdownHandlerIntegrationTest.java

```diff
@@ -32,7 +32,7 @@ public class ShutdownHandlerIntegrationTest {
         .withExposedPorts(5672)
         .withNetwork(COMMON_NETWORK);
     @Container
-    private static final ToxiproxyContainer TOXI_PROXY = new ToxiproxyContainer("shopify/toxiproxy:2.1.0")
```

Comment:
`shopify/toxiproxy:2.1.0` does not work for M1 chip locally, `shopify/toxiproxy:2.1.4` seems to be fine.